docs/examples/umbral_simple_api.py

```diff
@@ -17,14 +17,14 @@
 along with pyUmbral. If not, see <https://www.gnu.org/licenses/>.
 """
 
-#1
+# 1
```

Comment:
Can you remove these spaces? It makes demos more difficult cause you have to do the space and when you have one hand and the other has a mic, it's a bit annoying. :P